CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # SAS Viya Monitoring for Kubernetes
 
+## UNRELEASE
+
+* **Overall**
+
+* **Metrics**
+  * [FIX] - Deploy_monitoring_cluster script now correctly removes the v4m-kubelet service that is deployed by deploy_monitoring_cluster.
```

Comment:
A little too verbose...and incorrect (it is the remove_monitoring_cluster script that does the removing).  Maybe something like  "[FIX] - Remove v4m-kubelet service when removing cluster-level metric monitoring components"